services/web/server/src/simcore_service_webserver/version_control_db.py

```diff
@@ -490,3 +526,46 @@ async def get_snapshot_content(self, repo_id: int, commit_id: int) -> Dict:
                     return snapshot.content
 
         raise NotFoundError(name="snapshot for commit", value=(repo_id, commit_id))
+
+    async def get_workbench_view(self, repo_id: int, commit_id: int) -> Dict[str, Any]:
+        # FIXME: q&d fix!!!
+        # TODO: move to VersionControlRepositoryInternalAPI
```

Comment:
I am not too sure about this kind of comments... are you sure it's ready?

Reply:
I forgot to cleanup some of those